### PR TITLE
Track the version of AutoQC Loader that is pinging a Panorama QC folder

### PR DESCRIPTION
**Pull request by vagisha**
#### Rationale
The next version of AutoQC Loader will be sending the software version with the ping request.  

#### Changes
- Added a SoftwareVersion column to the AutoQCPing table
- Include the software version, when available,  in the AutoQC tooltip
- Updated test

### CODE REVIEW

File: resources/schemas/dbscripts/sqlserver/targetedms-23.005-23.006.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE targetedms.AutoQCPing ADD SoftwareVersion VARCHAR(100);
```
Comment (labkey-jeckels) [inline resources/schemas/dbscripts/sqlserver/targetedms-23.005-23.006.sql]:
```suggestion
ALTER TABLE targetedms.AutoQCPing ADD SoftwareVersion NVARCHAR(100);
```

File: webapp/TargetedMS/js/QCSummaryPanel.js [hunk 2]
```diff
@@ -223,6 +223,10 @@ Ext4.define('LABKEY.targetedms.QCSummary', {
         {
             var modifiedFormatted = Ext4.util.Format.date(new Date(autoQC.modified), LABKEY.extDefaultDateTimeFormat || 'Y-m-d H:i:s');
             content = autoQC.isRecent ? 'AutoQC pinged recently on ' + modifiedFormatted : 'AutoQC last pinged on ' + modifiedFormatted;
+            if (autoQC.softwareVersion)
+            {
+                content += "<br/>Version: " + autoQC.softwareVersion;
```
Comment (labkey-jeckels) [inline webapp/TargetedMS/js/QCSummaryPanel.js]: The version should be HTML encoded.

**Review by labkey-jeckels — APPROVED** [review]
Some small requests but looks good overall.

File: webapp/TargetedMS/js/QCSummaryPanel.js (hunk 2 above)
Comment (vagisha) [reply webapp/TargetedMS/js/QCSummaryPanel.js]: Done

**Pull request merged**